odoo/addons/base/res/res_company_view.xml

```diff
@@ -28,18 +28,18 @@
                                         <field name="zip" placeholder="ZIP" class="o_address_zip"/>
                                         <field name="country_id" placeholder="Country" class="o_address_country" options='{"no_open": True}'/>
                                     </div>
-                                    <field name="website" widget="url" placeholder="e.g. www.odoo.com"/>
-                                    <field name="report_footer" placeholder="e.g. Your Bank Accounts, one per line"/>
                                     <field name="report_header" placeholder="e.g. Global Business Solutions"/>
                                 </group>
                                 <group>
+                                    <field name="website" widget="url" placeholder="e.g. www.odoo.com"/>
                                     <field name="phone"/>
                                     <field name="email"/>
                                     <field name="vat"/>
                                     <field name="company_registry"/>
-                                    <field name="currency_id" options="{'no_create': True, 'no_open': True}" id="company_currency"/>
+                                    <field name="currency_id" options="{'no_create': True, 'no_open': True}" id="company_currency" invisible="1"/>
```

Comment:
Didn't we spoke about putting it in group_no_one ?

Reply:
FP specification was to remove it from view. But if I remember well, it 's used by something else.